qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -265,35 +265,32 @@ def export_to_file(
 
 
 def convert_dds_to_driven_control(
-    dynamic_decoupling_sequence: DynamicDecouplingSequence = None,
-    maximum_rabi_rate: float = 2 * np.pi,
-    maximum_detuning_rate: float = 2 * np.pi,
-    minimum_segment_duration: float = 0.0,
-    **kwargs,
+    dynamic_decoupling_sequence: DynamicDecouplingSequence,
+    maximum_rabi_rate: float,
+    maximum_detuning_rate: float,
+    minimum_segment_duration: Optional[float] = None,
```

Comment:
Hm, IMO it would be simpler just to default this to zero, and even to allow the user to pass zero -- zero seems like a reasonably valid value to me.

Reply:
SG, change back to zero.